internal/bminventory/inventory.go

```diff
@@ -2018,14 +2018,29 @@ func (b *bareMetalInventory) generateClusterInstallConfig(ctx context.Context, c
 		return errors.Wrapf(err, "failed to get install config for cluster %s", cluster.ID)
 	}
 
+	installerReleaseImageOverride := ""
```

Comment:
maybe it make more sense to move the definition before the next if? it will make the definition clearer.